### PR TITLE
Update private registry docs for CAPI images

### PR DESCRIPTION
**Pull request by bobdonat**
Update private registry docs for making CAPI images public.

### CODE REVIEW

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md [hunk 1]
```diff
@@ -195,6 +195,32 @@ Load the product images into your private registry.
...
    ```
 </div>
+{{< /clipboard >}}
+
+      * All the Cluster API (CAPI) images.
+{{< clipboard >}}
+<div class="highlight">
+
+   ```
+   $ cat ${DISTRIBUTION_DIR}/manifests/verrazzano-bom.json | jq -r '.components[].subcomponents[] | select(.name == "capi-cluster-api") | .images[] | "\(.image):\(.tag)"'
```
Comment (fredtibbitts) [inline content/en/docs/setup/private-registry/private-registry-full-distribution.md:205]: You can collapse these 3 jq statements into 1 with:
```
jq -r '.components[].subcomponents[] | select(.name | startswith("capi-")) | .images[] | "\(.image):\(.tag)"'
```

**Review by rosemarymarano — APPROVED** [review]
LGTM

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md (hunk 1 above)
Comment (bobdonat) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md:205]: that is better.  i will collapse to one jq

**Review by fredtibbitts — APPROVED** [review]

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md (hunk 1 above)
Comment (bobdonat) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md]: fixed

**Pull request merged**